pkg/hub/managedclusterset/default_managedclusterset_controller.go

```diff
@@ -110,6 +114,13 @@ func (c *defaultManagedClusterSetController) sync(ctx context.Context, syncCtx f
 func (c *defaultManagedClusterSetController) syncDefaultClusterSet(ctx context.Context, originalDefaultClusterSet *clusterv1beta1.ManagedClusterSet) error {
 	defaultClusterSet := originalDefaultClusterSet.DeepCopy()
 
+	// if the annotation has set to disable, default clusterset controller will not work.
+	if hasAnnotation(defaultClusterSet, defaultClusterSetAnnotationKey, defaultClusterSetAnnotationDisabledValue) {
+		c.eventRecorder.Eventf("DefaultManagedClusterSetDisabled", "The DefaultManagedClusterSet is disabled by user")
```

Comment:
I think it will generate a lot of event if annotation is set. We may only set a log with v(4)